### PR TITLE
Pass operator logos to map popup

### PR DESCRIPTION
**Pull request by amy-corson-ibigroup**
Pass the `getEntityPrefix` function that brings transit operator logos into `map-popup` by way of the tile overlay.

image

### CODE REVIEW

**Comment by binh-dam-ibigroup** [comment]
> Pass the `getEntityPrefix` function that brings transit operator logos into `map-popup` by way of the tile overlay.
> 
> image

Could you add a story that illustrates using `getEntityPrefix`, please?

File: packages/map-popup/src/MapPopup.story.tsx [hunk 1]
```diff
@@ -64,13 +67,33 @@ const FLOATING_CAR = {
...
   <MapPopupContents
     entity={STOP}
     setLocation={action("setLocation")}
     setViewedStop={action("setViewedStop")}
   />
 );
+export const StopEntitywithEntityPrefix = (): JSX.Element => (
+  <MapPopupContents
+    entity={STOP}
+    getEntityPrefix={() => getEntityPrefixExample(STOP)}
```
Comment (binh-dam-ibigroup) [inline packages/map-popup/src/MapPopup.story.tsx:92]: I think, per `packages/map-popup/src/index.tsx`, this attribute can be simplified to `getEntityPrefix={getEntityPrefixExamnple}`.

**Review by binh-dam-ibigroup — APPROVED** [review]
Just one story tweak, and remember to resolve merge conflicts.

**Review by josh-willis-arcadis — APPROVED** [review]
Changes look great!

**Pull request merged**